packages/cli/src/commands/init/init.ts

```diff
@@ -224,13 +206,12 @@ export default (async function initialize(
    */
   const version: string = minimist(process.argv).version || DEFAULT_VERSION;
 
-  const directoryName = path.relative(root, options.directory || projectName);
+  const projectPath = path.resolve(root, options.directory || projectName);
 
   try {
-    await createProject(projectName, directoryName, version, options);
+    await createProject(projectName, projectPath, version, options);
 
-    const projectFolder = path.join(root, directoryName);
```

Comment:
This results in "Run instructions" having absolute paths. I don't mind, but remember we had a little back and forth in this area :D

Reply:
I'll just go back to what was here before to avoid any unnecessary discussions again. XD